docs/elements/context-classes-vs-traits.html

```diff
@@ -0,0 +1,50 @@
+<article-header>Context Classes vs. Traits</article-header>
+<article-paragraphs>
+    <p>
+        In this package, both context classes and traits are utilized to extend the functionality and modularity of your
+        Behat tests. While context classes serve as the primary structure for defining test behaviors, traits allow you
+        to encapsulate and reuse common test logic across different contexts.
+    </p>
+    <p>
+        Traits provide predefined methods that can be included in your context classes to perform specific actions. By
+        using traits, you can avoid code duplication and ensure consistency across your test scenarios. For example, the
+        <code class="underline">HttpResponse</code> trait includes methods for asserting the status and content of HTTP
```

Comment:
At first I thought it's clickable, but not. Maybe we should style it in another way?